src/pytest_rich/capture.py

```diff
@@ -0,0 +1,66 @@
+import re
+from datetime import datetime
+from pathlib import Path
+
+from rich.console import Console
+
+
+def save_terminal_output(console: Console, arg: str) -> None:
+    """
+    Save terminal output to file.
+
+    Args:
+        console (Console): Rich console.
+        arg (str): Argument to parse.
+    """
+    try:
+        filename = _get_filename_from_arg(arg)
+    except ValueError as e:
+        console.print(f"[red]Error saving terminal output: {e}[/red]")
+        return
+
+    func_name = "text" if filename.endswith(".txt") else filename.split(".")[-1]
```

Comment:
Minor suggestion: I think it would be better for `_get_filename_from_arg` to return a tuple `(filename, file_type)`, so the code here can use it directly.